docs/topics/whatsnew18.md

```diff
@@ -182,46 +182,46 @@ the other is for `AndroidSourceSets`:
 To address these and other existing issues, we've introduced a new Android source set layout. 
 Here are some of the key differences between the two layouts:
 
-* __KotlinSourceSet naming schema__
+#### KotlinSourceSet naming schema
 
-  | Current source set layout | New source set layout           |
-  |---------------------------|---------------------------------|
-  | `targetName` + `AndroidSourceSet.name` | targetName + AndroidVariantType |
+| Current source set layout              | New source set layout           |
+|----------------------------------------|---------------------------------|
+| `targetName` + `AndroidSourceSet.name` | targetName + AndroidVariantType |
 
-  `{AndroidSourceSet.name}` maps to `{KotlinSourceSet.name}` as follows:
+`{AndroidSourceSet.name}` maps to `{KotlinSourceSet.name}` as follows:
 
-  | `AndroidSourceSet.name` | `KotlinSourceSet.name` in current source set layout | `KotlinSourceSet.name` in new source set layout |
-  |-------------------------|-----------------------------------------------------|-------------------------------------------------|
-  | `main`                  | `androidMain`                                       | `androidMain`                                   |
-  | `test`                  | `androidTest`                                       | `androidUnitTest`                        |
-  | `androidTest`           | `androidAndroidTest`                         | `androidInstrumentedTest`                |
+| `AndroidSourceSet.name` | `KotlinSourceSet.name` in current source set layout | `KotlinSourceSet.name` in new source set layout |
+|-------------------------|-----------------------------------------------------|-------------------------------------------------|
+| main                    | androidMain                                         | androidMain                                     |
+| test                    | androidTest                                         | android<b>Unit</b>Test                          |
```

Comment:
Unfortunately, bold is not supported in any code. So I had to remove all "code" signs.